src/Sylius/Component/Review/Model/Reviewable.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Component\Review\Model;
+
+use Doctrine\Common\Collections\Collection;
+
+/**
+ * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
+ */
+interface Reviewable
```

Comment:
Symfony2 codings standards (or PSR-2 too, I'm not sure) says, that there must be `Interface` suffix after interface name.